README.md

```diff
@@ -107,23 +107,17 @@ See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a
 The repository has the following packages (sub projects):
 
 - [attestation-service](packages/attestation-service) - service run by validators on the Celo network to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network
-- [blockchain-api](packages/blockchain-api) - service that uses Blockscout to present view of transactions by account for Celo Wallet activity feed
 - [celotool](packages/celotool) - scripts for deploying and managing testnets
 - [cli](packages/cli) - tool that uses ContractKit to interact with the Celo protocol ([docs](https://docs.celo.org/command-line-interface/introduction))
-- [contractkit](packages/contractkit) - library to help developers and validators interact with the protocol and it's smart contracts ([docs](https://docs.celo.org/celo-sdk/contractkit))
-- [dappkit](packages/dappkit) - set of functions for mobile DApps to work with the wallet app (ex. sign transactions and access the user's account) ([docs](https://docs.celo.org/celo-sdk/dappkit))
 - [dev-utils](packages/dev-utils) - a utils package for use as a dev dependency
 - [docs](packages/docs) - technical documentation for the Celo project ([live](https://docs.celo.org/))
 - [faucet](packages/faucet) - faucet deployment configuration ([live](https://celo.org/build/faucet))
 - [helm-charts](packages/helm-charts) - templatized deployments of entire environments to Kubernetes clusters
-- [mobile](packages/mobile) - Android wallet app for the Celo platform ([docs](https://docs.celo.org/getting-started/using-the-mobile-wallet), [live](https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores))
-- [notification-service](packages/notification-service) - service for managing push notifications for Celo Wallet
 - [protocol](packages/protocol) - identity, stability and other smart contracts for the Celo protocol ([docs](https://docs.celo.org/celo-codebase/protocol))
-- [react-components](packages/react-components) - generalized react native components
-- [reserve-site](packages/reserve-site) - website for ([Celo Reserve.org](https://celoreserve.org/))
+- [sdk](packages/sdk)
+- [contractkit](packages/sdk/contractkit) - library to help developers and validators interact with the protocol and its smart contracts ([docs](https://docs.celo.org/celo-sdk/contractkit))
```

Comment:
Oh i just noticed that link  for docs is broken

https://docs.celo.org/developer-guide/contractkit

really a good thing you are bringing our attention to this

Reply:
Fixed!